dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -79,12 +90,58 @@ def self.get_lesson_s3_name(script_level)
     [code, version]
   end
 
-  private def get_fake_openai_evaluations(rubric, understanding_s: 'Extensive Evidence')
-    rubric.learning_goals.map do |learning_goal|
-      {
-        'Key Concept' => learning_goal.learning_goal,
-        'Grade' => understanding_s
-      }
+  private def read_file_from_s3(lesson_s3_name, key_suffix)
+    bucket = 'cdo-ai'
+    key = "teaching_assistant/lessons/#{lesson_s3_name}/#{key_suffix}"
+    s3_client.get_object(bucket: bucket, key: key)[:body].read
+  end
+
+  private def read_examples(lesson_s3_name)
+    bucket = 'cdo-ai'
+    prefix = "teaching_assistant/lessons/#{lesson_s3_name}/examples/"
+    response = s3_client.list_objects_v2(bucket: bucket, prefix: prefix)
```

Comment:
I wonder if it makes sense to consider caching these files where the job worker is somehow, than to negotiate and download these files for every student submission when they are not likely to change.

Reply:
Perhaps we can revisit this more later.

Reply:
yeah, I agree that we should do this, but also think we can get away without it for the pilot. tracking it here along with a link to one way of doing it: https://codedotorg.atlassian.net/browse/AITT-250